### DISCUSSION TITLE
DeprecationWarning

### DISCUSSION THREAD

**Discussion by pbryan** [aio-libs/aiobotocore#908] [General]
I'm noticing a deprecating warning, and wondering if this is a transitive dependency issue or something I may have misconfigured.

```
  /home/pbryan/Projects/fondat-aws/.venv/lib/python3.10/site-packages/botocore/httpsession.py:62: DeprecationWarning: ssl.PROTOCOL_TLS is deprecated
    context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
```
I'm using aiobotocore 2.0.1 and aiohttp 3.8.1.

Does this ring any bells?

**Comment by asvetlov** [comment]
I guess you can just ignore this warning until the next aiobotocore release (with pinned botocore version that is actually the source of the warning).
